memoize/wrapper.pyi

```diff
@@ -0,0 +1,17 @@
+from typing import Callable, TypeVar, overload
+
+from memoize.configuration import CacheConfiguration
+from memoize.invalidation import InvalidationSupport
+
+FN = TypeVar('FN', bound=Callable)
+
+
+@overload
+def memoize(*, configuration: CacheConfiguration = None,
```

Comment:
`configuration` and `invalidation` are optional and should be marked accordingly (Optional from typing).